login/service.go

```diff
@@ -624,6 +624,9 @@ func (keycloak *KeycloakOAuthProvider) CreateOrUpdateIdentityInDB(ctx context.Co
 		if identity.User.Cluster == "" {
 			identity.User.Cluster = configuration.GetOpenShiftClientApiUrl()
 		}
+		if identity.User.FeatureLevel == "" {
```

Comment:
just curious: what's the benefit of setting the default value here instead of using a `default` constraint in the column of the database table ?

Reply:
because we already have it in the database, and it still doesn't work :)

Reply:
ah of course! the value is `<empty>` but not `NULL`, so the `default` is not applied :/